pkgs/desktops/gnome-2/desktop/mail-notification/default.nix

```diff
@@ -0,0 +1,87 @@
+{ stdenv, fetchurl, fetchFromGitHub, pkgconfig, glib, gnome2, dbus_glib, gmime, libnotify, libgnome_keyring, openssl, cyrus_sasl, gnonlin, sylpheed, gob2, gettext, intltool, libxml2, hicolor_icon_theme, tango-icon-theme }:
+
+stdenv.mkDerivation rec {
+  rev = "9ae8768";
+  version = "5.4";
+  name = "mail-notification-${version}";
+
+  src = fetchFromGitHub {
+    inherit rev;
+    owner = "epienbroek";
+    repo = "mail-notification";
+    sha256 = "1slb7gajn30vdaq0hf5rikwdly1npmg1cf83hpjs82xd98knl13d";
+  };
+
+   buildInputs = [ pkgconfig glib dbus_glib gmime libnotify libgnome_keyring openssl cyrus_sasl gnonlin sylpheed gob2 gettext intltool gnome2.GConf gnome2.libgnomeui dbus_glib gmime libnotify gnome2.gnome_keyring gnome2.scrollkeeper libxml2 gnome2.gnome_icon_theme hicolor_icon_theme tango-icon-theme ];
+
+  prePatch = ''
+    sed -i  -e '/jb_rule_set_install_message/d' -e '/jb_rule_add_install_command/d' jbsrc/jb.c || die
```

Comment:
Where does `die()` come from? I'd expect `set -e` in the generic setup script would be sufficient, is it not?